pow-migration/src/monitor.rs

```diff
@@ -72,7 +72,7 @@ pub fn generate_online_tx(validator: String) -> OutgoingTransaction {
         to: Address::burn_address().to_user_friendly_address(),
         value: 1, //Lunas
         fee: 0,
-        data: Some("online".to_string()),
+        data: Some(Blake2bHash::default().to_hex()),
```

Comment:
Default hash is 64 0s long. A lot of unnecessary data. You can get the bytes of a string with `"online".to_string().bytes()`, which you can then put as the data.

Reply:
I thought about keep using "online" but then realized that later in the process what we are doing is sending a hash (genesis hash), so I thought it was ok to send the default hash as an indication of "we have not computed a hash yet" 
But since we are not doing anything useful with this field, I guess "online" or anything is fine

Reply:
The 0-hash is _theoretically_ still a valid genesis block hash :D